wyvern/wyvern_request.py

```diff
@@ -43,6 +44,7 @@ class WyvernRequest:
     events: List[Callable[[], List[LoggedEvent[Any]]]]
 
     feature_map: FeatureMap
+    model_score_map: Dict[str, Dict[Identifier, float]]
```

Comment:
Why don't we start inserting values to this in the model component?

Reply:
Yeah that'll be the plan in the next PR which is WIP.  this one is more for cleaning up some code and setting up stuff to do it. maybe it'll be easier to see it through when i have the next PR.